src/renderer/Presentational/ContentMultipleClients/ContentMultipleClients.tsx

```diff
@@ -175,37 +213,30 @@ const ContentMultipleClients = (props: {
       <HeaderMetrics {...(nodeOverview as NodeOverviewProps)} />
       <HorizontalLine type="content" />
       {renderPrompt()}
-      <div className={sectionTitle}>Ethereum Clients</div>
+      <div className={sectionTitle}>Clients</div>
       <div className={clientCardsContainer}>
         {clients.map((client) => {
-          return <ClientCard {...client} />;
+          return (
+            <ClientCard
+              {...client}
+              onClick={() => {
+                console.log(
+                  'ContentMultipleClients client on click!',
+                  client.id,
+                );
+                dispatch(updateSelectedNodeId(client.id));
+                setTimeout(() => {
+                  navigate('/main/node');
+                }, 500);
```

Comment:
Added a delay to navigate because NodeScreen can't handle a node change properly here after NodeScreen renders